Add files via upload

FangTang999 opened the pull request:
Add the dataset of Washington metrorail network

jdlph commented:
Thank you for opening up this pull request! Would it be possible for you to change the destination branch to dev? All potential changes/updates shall go to the dev branch before merging into master for release. This is considered as one of best practices.

FangTang999 commented:
Hi, Peiheng

I changed the destination branch to dev, and deleted the branch I created
earlier. If there is anything I need to do, please let me know.

Best,
Fang(Alicia) Tang

Peiheng Li ***@***.***> 于2021年6月10日周四 上午8:14写道：

> Thank you for opening up this pull request! Would it be possible for you
> to change the destination branch to dev? All potential changes/updates
> shall go to the dev branch before merging into master for release. This is
> considered as one of best practices.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jdlph/Path4GMNS/pull/5#issuecomment-858184384>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJGLCZKW2ZYHXUBPIBMP6WLTR77URANCNFSM46KF22AA>
> .
>
